src/tests/ftest/pool/multi_server_create_delete_test.py

```diff
@@ -80,28 +80,23 @@ def test_create(self):
         # expected result of the test
         expected_for_param = []
 
-        modelist = self.params.get("mode", '/run/tests/modes/*')
-        mode = modelist[0]
-
-        expected_for_param.append(modelist[1])
-
-        uidlist = self.params.get("uid", '/run/tests/uids/*')
-        if uidlist[0] == 'valid':
-            uid = os.geteuid()
-        elif uidlist[0] == 'other_user':
-            uid = getpwnam('nfsnobody')[2]
+        userlist = self.params.get("user", '/run/tests/users/*')
+        if userlist[0] == 'valid':
+            user = os.getlogin()
+        elif userlist[0] == 'other_user':
+            user = 'nfsnobody'
```

Comment:
Can this be simplified by just placing `nfsnobody` in the yaml file, then the user could assigned like so:
`user = os.getlogin() if userlist[0] == 'valid' else userlist[0]`